graylog2-web-interface/src/views/types.ts

```diff
@@ -500,6 +509,7 @@ declare module 'graylog-web-plugin/plugin' {
     'views.hooks.copyPageToDashboard'?: Array<CopyParamsToView>;
     'views.hooks.removingWidget'?: Array<RemovingWidgetHook>;
     'views.overrides.widgetEdit'?: Array<React.ComponentType<OverrideProps>>;
+    'views.sidebar.links'?: Array<SidebarLink>;
```

Comment:
I think it would be more useful to have something like `views.sidebar.topSection` or `searchSystemSection` and the Items can have some flag `isLink`. Having just `links` means that there will be just links and there might be multiple links and I'm not sure that we will add more different links (except something similar to DWH). Also if we would decided to have another section with some links it might be confusing.

Reply:
I tried to improve the name.